license-details/README.md

```diff
@@ -0,0 +1,139 @@
+# Introduction
+
+The P4 compiler `p4c` is a leading project of the P4 Consortium, and
+used by several companies as the front end of P4 compilers for a
+variety of hardware targets.  Most of these companies develop
+significant proprietary back end code, and the resulting binary
+executables are released, with no copyleft obligation for those
+companies to release their back end source code.
+
+This is allowed, because the `p4c` front source files are licensed
```

Comment:
Did you mean "frontend source files" or did you mean "source code" or anything else?

Reply:
I went ahead and made that replacement in commit 7 of this PR.  Feel free to suggest other wordings if that is not clear enough.